supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc

```diff
@@ -1047,14 +1068,25 @@ Use the name of the tab when you refer to it, for example, "the *Storage* tab".
 [discrete]
 [[rpm]]
 ==== image:images/yes.png[yes] RPM (noun)
-*Description*: _RPM_ is the recursive initialism for the _RPM Package Manager_. RPM manages files in the RPM format, known as RPM packages. RPM packages are known informally as `rpm` files, but this informal usage is not used in Red Hat documentation to avoid confusion with the command name. Files in RPM format are referred to as "RPM packages".
+*Description*: _RPM_ is the recursive acronym for the _RPM Package Manager_. RPM manages files in the RPM format, known as RPM packages. RPM packages are known informally as rpm files, but this informal usage is not used in Red Hat documentation to avoid confusion with the command name. Files in RPM format are referred to as RPM packages.
```

Comment:
* Initialism was actually the correct term here
* I think the quotes you removed were correct
* If the backticks are being removed from `rpm`, then I think it needs to be capitalized as RPM files

Reply:
Thank you for catching this. Applying the feedback and rewording a bit.